packages/cli/src/lib/cmds/build.ts

```diff
@@ -55,6 +57,20 @@ class Build {
       private prompt: Prompt = new InquirerPrompt()) {
   }
 
+  /**
+   * Checks if the twa-manifest.json file has been changed since the last time the project was generated.
+   */
+  async checkManifest(manifestFile: string): Promise<boolean> {
```

Comment:
Can we rename this to make it a bit clearer what the return type signifies? Something like `hasManifestChanged`?

Reply:
done.